java/jdbc/src/main/java/com/flipkart/vitess/jdbc/VitessConnection.java

```diff
@@ -247,10 +247,7 @@ public void setReadOnly(boolean readOnly) throws SQLException {
                 Constants.SQLExceptionMessages.METHOD_CALLED_ON_OPEN_TRANSACTION);
         }
 
-        if (readOnly) {
-            throw new SQLFeatureNotSupportedException(Constants.SQLExceptionMessages.READ_ONLY);
-        }
-        this.readOnly = false;
+        this.readOnly = readOnly;
```

Comment:
Can you please leave a TODO note in the javadoc comment that this feature is actually not implemented and the call would be a no-op?

This will mitigate some of the confusion when users find out that the feature is not working as expected.